int64 center_x
int64 center_y
---
geometry_msgs/Point centroid_point